src/core/markdown/templates/custom-object-template.ts

```diff
@@ -23,6 +23,9 @@ export const customObjectTemplate = `
 **Type**
 
 *{{fieldType}}*
+{{#each pickListValues}}
```

Comment:
I'm thinking that instead of looping through the picklist values, we will want to make this its own section, and only render it if there are values to display (since not all fields will be picklists).

I like how Salesforce documentation does it, where they have the "Possible values are" as the heading

![image](https://github.com/user-attachments/assets/163946f7-2fc0-4646-8f92-9a3a0a9f99a3)

See my comment below in the `src/core/renderables/types.d.ts` file with the details about the changes that would need to be made to the type to accomplish this